neuro-sdk/src/neuro_sdk/_config.py

```diff
@@ -59,7 +59,7 @@ class _ConfigData:
     url: URL
     admin_url: Optional[URL]
     version: str
-    cluster_name: str
+    cluster_name: Optional[str]
```

Comment:
Please reflect the type change in https://github.com/neuro-inc/neuro-cli/blob/master/neuro-sdk/docs/config_reference.rst#L50

Reply:
This type didn't change. It is internal dataclass.